src/Context/contextTypes.ts

```diff
@@ -0,0 +1,13 @@
+import {ReactNode} from "react";
+import {MealsCountParams} from "../components";
+
+interface IContextTypes{
+    onClick: (data: MealsCountParams) => void;
+}
+
+interface IProviderTypes{
+    children: ReactNode
```

Comment:
To avoid necessity to define children you could use PropsWithChildren<> generic type from react